migrations/schema.sql

```diff
@@ -135,6 +142,20 @@ CREATE EXTENSION IF NOT EXISTS pgjwt WITH SCHEMA extensions;
 COMMENT ON EXTENSION pgjwt IS 'JSON Web Token API for Postgresql';
 
 
+--
```

Comment:
schema changes seem unrelated to u18 concerns?

Reply:
Yup, those were likely generated by a previous merge commit.
Fixed by merging `develop` into this branch